docs/guides/android_chiptool_building.md

```diff
@@ -0,0 +1,87 @@
+# Building Android CHIPTool
+
+Android CHIPTool is an application for Android for commissioning and controlling
+CHIP accessory devices.
+
+CHIPTool offers the following features:
+
+-   Scan a CHIP QR code and display payload information to the user
+-   Read the NFC tag containing CHIP onboarding information
+-   Commission a CHIP device
+-   Send echo requests to the CHIP echo server
+-   Send on/off cluster requests to a CHIP device
+
+<hr>
+
+-   [Source files](#source)
+-   [Requirements for building](#requirements)
+    -   [ABIs and TARGET_CPU](#abi)
+-   [Building Android CHIPTool](#building)
+
+<hr>
+
+<a name="source"></a>
```

Comment:
For my own knowledge: whad to these empty links do? Is there some docmentation about this?

Reply:
This is an html-like anchor for the header. It is used for creating the table of contents, which cannot be automatically generated in pure markdown. There are several ways you can create links to headers in markdown. I didn't find any official or officialish documentation, but this thread documents a number of them: https://stackoverflow.com/questions/6695439/how-to-link-to-a-named-anchor-in-multimarkdown